src/harness/ascii_tables.h

```diff
@@ -0,0 +1,893 @@
+#ifndef DETHRACE_ASCII_TABLES_H
```

Comment:
This file was generated with the attached python script + json files: [ascii-codegen.zip](https://github.com/dethrace-labs/dethrace/files/11392696/ascii-codegen.zip)